crates/cargo-wdk/README.md

```diff
@@ -0,0 +1,101 @@
+
+## Installation
+
+To install cargo-wdk, you need to have Rust and Cargo installed on your system. 
+You can install Rust and Cargo by following the instructions on the [Rust website](https://www.rust-lang.org/tools/install).
+
+Once you have Rust and Cargo installed, you can install cargo-wdk using the cargo install command in one of the following ways:
+
+### Install by cloning the source
+    - Clone the windows-drivers-rs repository.
+    - Navigate to crates\cargo-wdk directory.
+    - Run the following command:
+        ```pwsh
+        cargo install --path .
+        ```
+
+### Install by specifying Git repository
+    - Run the following command:
+        ```pwsh
+        cargo install --git https://github.com/microsoft/windows-drivers-rs.git --bin cargo-wdk
+        ```
+
+The install command compiles the `cargo-wdk` binary and copies the compiled binary to Cargo's bin directory (%USERPROFILE%.cargo\bin).
+
+You can test the installation by running the following command:
+```pwsh
+cargo wdk --version
+```
+
+For help on usage, run the command:
+```pwsh
+cargo wdk --help
+```
+
+## Installing WDK
+
+`cargo-wdk` builds the drivers using the WDK. Please ensure that the WDK is installed on the development system.
+The recommended way to do this is to [enter an eWDK developer prompt](https://learn.microsoft.com/en-us/windows-hardware/drivers/develop/using-the-enterprise-wdk#getting-started)
+
+## Usage
+
+1. Use the `new` command to create a new Rust driver project - 
+    ```pwsh
+    cargo wdk new [OPTIONS] [DRIVER_PROJECT_NAME]
+    ```
+    
+    Example Usage:
+    ```pwsh
+    cargo wdk new sample_driver UMDF
+    ```
+    ```pwsh
+    cargo wdk new sample_driver --driver-type KMDF
+    ```
+
+    Use `--help` for more information on arguments and options
+
+2. Use the `build` command to build and package driver projects.
+    ```pwsh
+    cargo wdk build [OPTIONS]
+    ```
+    
+    Example Usage: 
+    * Navigate to the project/workspace root and run - 
+    ```pwsh 
+    cargo wdk build 
+    ```
+    ```pwsh 
+    cargo wdk build --cwd /path/to/project
+    ```
+
+    Use `--help` for more information on arguments and options
+
+## Features
+
+- `new` command uses templates in `crates/cargo-wdk/templates` directory to scaffold the project appropriately.
+
+- `build` command can be used for individual projects and workspaces. Workspaces with both driver and non-driver members are also supported.
+
+    1. For individual projects, a signed driver package, including a `WDRLocalTestCert.cer` file, will be generated at `target/<Cargo profile>/_package`. If a specific target architecture was specified, the driver package will be generated at `target/<target architecture>/<Cargo profile>/_package`
+    2. For workspaces, 
+        - Non-driver members are compiled (using standard cargo build), 
+        - Driver members are built and packaged. The final driver packages will be generated in the workspace's `/target/` directory.
+
```

Comment:
I feel the "Features" section should be moved up and placed right before the "Installation" section because it would be helpful to explain what the tool does before we talk about how to install it.